### PR TITLE
Home page with assets

### PR DESCRIPTION
**Pull request by mamadOuologuem**

### CODE REVIEW

File: src/components/MenuBar.js [hunk 1]
```diff
@@ -35,17 +35,36 @@ const useStyles = makeStyles({
     width: 'fit-content',
     opacity: 1,
   },
+  button: props => ({
```
Comment (bencpeters) [inline src/components/MenuBar.js]: Let's destructure `props` here.

File: src/components/home/MapSection.js [hunk 2]
```diff
@@ -3,23 +3,26 @@ import { Grid, Typography } from '@material-ui/core';
...
 
 const useStyles = makeStyles({
-  mainContainer: {
-    height: '680px',
-    backgroundImage: `url(${map})`,
+  title2: { margin: '100px 0px -250px' },
+  image: {
+    position: 'relative',
+    width: 'calc(100vw - 8px)',
+    left: 'calc(-1 * (100vw - 100%) / 2)', // to centered the div
```
Comment (bencpeters) [inline src/components/home/MapSection.js]: Didn't we change this in the last PR?

Comment (mamadOuologuem) [reply src/components/home/MapSection.js]: No it is not the same this is for the map image (yes it is a copy paste)

**Comment by mamadOuologuem** [comment]
> Mostly looks good.
> 
> I'm not sure we want to have these larger PNG files as major banner images on the homepage though. What does the size look like if we have the banner and map files as fairly high res JPEGs? Was there a reason you switched to PNGs?

Indeed they has large sizes. I changed because in JPEG the images was a bit blurred (but I will try the put them back to JPEG and see how it looks like)

**Comment by mamadOuologuem** [comment]
> > Mostly looks good.
> > I'm not sure we want to have these larger PNG files as major banner images on the homepage though. What does the size look like if we have the banner and map files as fairly high res JPEGs? Was there a reason you switched to PNGs?
> 
> Indeed they has large sizes. I changed because in JPEG the images was a bit blurred (but I will try the put them back to JPEG and see how it looks like)

it seems okay for the banner but the map's color is changing a bit when I switched to jpg

**JPG**

VERSUS **PNG**

**Comment by bencpeters** [comment]
What program are you using to make the JPEGs? And what compression settings? You can probably play with the color profile a bit to figure out why it's changing...

What is the file size of the JPEG versions of the images?

**Pull request merged**